src/content/resources/language/evolution.md

```diff
@@ -42,6 +42,36 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.8
```

Comment:
Consider also mentioning the [formatting changes](https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#:~:text=made%20a%20number%20of%20fixes%20in%20response%20to%20that) that are [language versioned](https://dart.dev/resources/language/evolution#language-versioning) for 3.8.

https://github.com/dart-lang/site-www/pull/6605 also mentions them.

Reply:
Added this and also updated the formatting of this section. Take a close look.